lib/yarv/frame.rb

```diff
@@ -28,5 +31,13 @@ def get_local(index)
     def set_local(index, value)
       @locals[index] = value
     end
+
+    def set_block(block)
+      @block = block
+    end
+
+    def execute_block(*arguments)
+      block.call(*arguments)
+    end
```

Comment:
@kddnewton how do you feel about this implementation? If I understand it's not the most accurate representation (the block should be in the same "locals" table) but was the most convenient to represent. 

I would be happy to refactor `locals` to it's own class in another PR.

Reply:
I think this works! We can work on locals later. Want to mark this as ready for review so CI runs?

Reply:
Done!